website/www/site/content/en/blog/adding-data-sources-to-sql.md

```diff
@@ -81,7 +81,7 @@ The `TableProvider` classes are under
 
 Our table provider looks like this:
 
-{{< highlight java >}}
+{{< highlight language="java" >}}
```

Comment:
not really, because when we have buttons leading to run the code or to the source code, there are 3 arguments there and not using variables when there are really long paths to those buttons above, will end with a lot of mistakes and the highlights will be unreadable